nekoyume/Assets/_Scripts/UI/Module/EventNotice/NoticeView.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Nekoyume.UI.Model;
+using TMPro;
+using UnityEngine;
+
+namespace Nekoyume.UI.Module
+{
+    public class NoticeView : MonoBehaviour
```

Comment:
It's component for the marked portion.
![image](https://user-images.githubusercontent.com/48484989/206069312-5f5b1f97-63cc-442d-b260-ce107d6272a3.png)